### DISCUSSION TITLE
The extension doesnt work

### DISCUSSION THREAD

**Discussion by SUMUKHA-PK** [James-Yu/LaTeX-Workshop#3155] [Q&A]
After install and pressing the `Open Preview` button it returns a `command 'vscode.previewHtml' not found` error.
Using the latest version.

**Comment by SUMUKHA-PK** [comment]
And `Recipe terminated with fatal error: spawn latexmk ENOENT.` for "running" it.

**Comment by tamuratak** [comment]
Fill out all the information required in the issue template. Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

**Comment by tamuratak** [comment]
When all of the above is done, this issue will be reopened.
